game-core/src/test/java/games/strategy/triplea/delegate/battle/casualty/AaCasualtySelectorTest.java

```diff
@@ -417,4 +419,606 @@ void hitsLessThanPlanesKillsAccordingToTheRolledDice() {
           is(List.of(planes.get(0), planes.get(2), planes.get(4))));
     }
   }
+
+  @Nested
+  class LowLuckCasualties {
+    private IDelegateBridge bridge;
+    @Mock private UnitType otherPlaneUnitType;
+    @Mock private UnitType otherPlaneMultiHpUnitType;
+    @Mock private UnitType powerfulAaUnitType;
+
+    @BeforeEach
+    void setupPlayers() {
+      when(hitPlayer.getName()).thenReturn("Hit Player");
+    }
+
+    @BeforeEach
+    void initializeGameData() {
+      gameData =
+          givenGameData()
+              .withDiceSides(6)
+              .withEditMode(false)
+              .withChooseAaCasualties(false)
+              .withLowLuck(true)
+              .build();
+      bridge = mock(IDelegateBridge.class);
+      when(bridge.getData()).thenReturn(gameData);
+
+      aaUnitType = new UnitType("aaUnitType", gameData);
```

Comment:
observation, the setup is very beefy, lots of test-state is established. If we add a few more tests like this it will be more worthwhile to invest in easier setup. Ideally a test case can have relatively minimal fixture code around it. Just noting that I'm not sure this pattern will scale super gracefully with a lot more tests.